osidb/models.py

```diff
@@ -2665,6 +2655,18 @@ def is_triage(self):
         # it is to be determined and implemented
         return False
 
+    @property
+    def is_contract_priority(self) -> bool:
+        """
+        check and return whether this affect module and update stream combination is contract
+        priority which is defined in PS constants repo in contract_priority.yml
+        """
+        # A multi-flaw tracker is assumed to be related only to a single module.
+        ps_module = self.affects.first().ps_module
+        return ContractPriority.objects.filter(
+            ps_module=ps_module, ps_update_stream=self.ps_update_stream
+        ).exists()
```

Comment:
I am thinking about the definition of the contract priority and I tent to suggest a simplification.

The thing is that when I give you an update stream I am also giving you the module implicitly without even mentioning it. There is a clear hierarchy in product definitions so that each update stream belongs to one specific module. Then however the module here is obviously an unnecessary extra information. The update stream is enough to determine the contract priority.

I know that in the PS constants there are both and I was suggesting to avoid the modules there as unnecessary from the tooling perspective but the opinion that it is better for readability from the human perspective won. Here in our code/DB however we could prefer machine over man :smile: WDYT?

Reply:
Fixed.